x-pack/plugins/monitoring/common/ccs_utils.ts

```diff
@@ -20,20 +16,32 @@ export function getConfigCcs(config: MonitoringConfig): boolean {
  *
  * @param  {Object} config The Kibana configuration object.
  * @param  {String} indexPattern The index pattern name
- * @param  {String} ccs The optional cluster-prefix to prepend.
+ * @param  {Array|String} ccs The optional cluster-prefixes to prepend. An array when passed from config
+ * and a string when passed from a request.  This is optional because the request could be empty if its not a remote cluster
  * @return {String} The index pattern with the {@code cluster} prefix appropriately prepended.
  */
-export function prefixIndexPattern(config: MonitoringConfig, indexPattern: string, ccs?: string) {
-  const ccsEnabled = getConfigCcs(config);
+export function prefixIndexPatternWithCcs(
```

Comment:
I changed the name to have "WithCcs" because previously this function also handled appending metricbeat.  Now it only does CCS so want to clarify that.